bindings/python/src/trainers.rs

```diff
@@ -864,8 +868,8 @@ mod tests {
         let py_bpe = py_trainer.get_as_subtype().unwrap();
         let gil = Python::acquire_gil();
         assert_eq!(
-            "tokenizers.trainers.BpeTrainer",
-            py_bpe.as_ref(gil.python()).get_type().name()
+            "BpeTrainer",
```

Comment:
This is odd, I'm not sure why.

cc @adamreichold @davidhewitt

Reply:
I think is due to https://github.com/PyO3/pyo3/pull/1152/files, i.e. support for building against the stable CPython API, i.e. the last bullet point in https://pyo3.rs/v0.16.2/migration.html#runtime-changes-to-support-the-cpython-limited-api

Reply:
But that class does have `#[pyclass(module = "tokenizers.trainers")]`
https://github.com/huggingface/tokenizers/blob/cd730594e9d6b6349e4b95bc522525d7e5a178a6/bindings/python/src/trainers.rs#L18

Reply:
Maybe something is messed by repeated specifying `name=Trainer`?